package.json

```diff
@@ -6,7 +6,9 @@
   "main": "dist/index.js",
   "module": "dist/index-esm.js",
   "scripts": {
-    "lint": "eslint . && echo 'Lint finished.'",
+    "lint": "npm run lint-js && npm run lint-md && echo 'Lint finished.'",
```

Comment:
When running npm scripts within npm scripts, it's handy to use `npm run -s` instead of `npm run` to reduce error output noise.

Reply:
It is now [silent](https://github.com/ericelliott/autodux/pull/51/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R9). 🙂